### PR TITLE
Patch importNode to work around IE11 bug.

### PR DESCRIPTION
**Pull request by kevinpschaaf**

### CODE REVIEW

**Comment by garlicnation** [comment]
This fixes #6! You're a magician, kevin.

**Comment by addyosmani** [comment]
If we're gated on the Spartan testing to land this, I can verify in the morning and land. Otherwise, LGTM.

**Comment by garlicnation** [comment]
@addyosmani There's no rush to release this right now, and I'd feel better if it were tested in spartan before merging.

**Comment by addyosmani** [comment]
:+1:

File: src/CustomElements/boot.js [hunk 1]
```diff
@@ -57,6 +78,7 @@ if (!window.wrap) {
   }
 }
 
+
```
Comment (azakus) [inline src/CustomElements/boot.js]: micronit: extra newline

**Comment by addyosmani** [comment]
I spoke to the IE team yesterday. Whilst Spartan builds aren't yet available, the IETP + EdgeHTML on give you the equal experience. Testing these changes there, I wasn't able to find any edge cases that broke.

I think we can get Dan's micronit addressed in a follow up :) @garlicnation landing so you're not still waiting for this one on the rest of the PR queue. 

Great work @kevinpschaaf!

**Pull request merged**